flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/operators/asyncprocessing/AbstractAsyncStateStreamOperatorV2.java

```diff
@@ -113,6 +120,26 @@ public final <T> ThrowingConsumer<StreamRecord<T>, Exception> getRecordProcessor
                         + " since this part is handled by the Input.");
     }
 
+    @Override
+    @SuppressWarnings("unchecked")
+    public <K, N> InternalTimerService<N> getInternalTimerService(
+            String name, TypeSerializer<N> namespaceSerializer, Triggerable<K, N> triggerable) {
```

Comment:
A fallback plan when isAsyncStateProcessingEnabled=false might be needed here as well.